### ISSUE TITLE
Click on file paths/lines/columns in Code Lens test error output to go there

### ISSUE THREAD

**Issue by willfaught** [Microsoft/vscode-go#1562]
If you use Code Lens to run a test, you see output like this:

```
Running tool: /usr/local/bin/go test -timeout 30s github.com/foo/bar -run ^TestNewBaz$

# github.com/foo/bar
./baz_test.go:31:52: cannot convert "1" (type string) to type BazInt
./baz_test.go:37:58: cannot convert "\"1\"" (type string) to type BazString
./baz_test.go:38:55: cannot convert "1" (type string) to type BazInt
```

It'd be great if we could click on the `./baz_test.go:31:52` parts like we can in panic traces to go straight to that file/line/column. Is that possible?

**Comment by willfaught** [comment]
Awesome! Thanks!

**Comment by ramya-rao-a** [comment]
The fix for this is now available in the latest update (0.6.78) to the Go extension
